Index: ogb/nodeproppred/.~lock.master.csv#
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ogb/nodeproppred/.~lock.master.csv# b/ogb/nodeproppred/.~lock.master.csv#
new file mode 100755
--- /dev/null	
+++ b/ogb/nodeproppred/.~lock.master.csv#	
@@ -0,0 +1,1 @@
+,hussein,hussein-XPS-15-9500,05.04.2022 13:25,file:///home/hussein/.config/libreoffice/4;
\ No newline at end of file
Index: ogb/nodeproppred/.~lock.master.csv#
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ogb/nodeproppred/.~lock.master.csv# b/ogb/nodeproppred/.~lock.master.csv#
new file mode 100755
--- /dev/null	
+++ b/ogb/nodeproppred/.~lock.master.csv#	
@@ -0,0 +1,1 @@
+,hussein,hussein-XPS-15-9500,05.04.2022 13:25,file:///home/hussein/.config/libreoffice/4;
\ No newline at end of file
